src/arrays/index.md

```diff
@@ -84,15 +84,14 @@ note: library: util
 ```
 
 As well as compiling `averages.rs` into a static library, `rustc` has also
-emitted some helpful notes which we'll need when linking the library into one
-executable.
+emitted some helpful notes needed for linking the library into one executable.
```

Comment:
Does "linking the library into an executable", still imply that we're linking into only one executable?
"linking the library into one executable" just reads strangely to me.